django_ses/views.py

```diff
@@ -282,10 +305,81 @@ def handle_bounce(request):
                     },
                 )
 
-                signals.delivery_received.send(
-                    sender=handle_bounce,
+                if signal_sender == 'handle_bounce':
+                    signals.delivery_received.send(
+                        sender=handle_bounce,
+                        mail_obj=mail_obj,
+                        delivery_obj=delivery_obj,
+                        raw_message=raw_json,
+                    )
+                else:
+                    signals.delivery_received.send(
+                        sender=handle_event,
+                        mail_obj=mail_obj,
+                        delivery_obj=delivery_obj,
+                        raw_message=raw_json,
+                    )
+            elif event_type == 'Send':
```

Comment:
I can't tell by reading the code but, should these new event types be prevented from going through `handle_bounce`? Seems like it could cause issues for existing uses?

Reply:
Yeah, you are right, I have missed this. If you get an `send` event in the previous implementation, it logs a `warning`. With current implementation, it will be handled and a signal will be send.

What about using the existing `handle_bounce` view as is and add `handle_event` as a separated view? We can add minor methods for each event type to avoid code repetition.

Reply:
I think that would be better.

Does it make sense to remove `handle_bounce`?

What do you think about logging a warning if users of the library import `handle_bounce`, so we can deprecate it in the next major release? That way we keep backwards compatible for now but warn people to prepare it will be removed.

Reply:
Agreed, removing `handle_bounce` eventually makes sense.
We may want to update `django-ses[bounce]` to `django-ses[events]` as well.